nservicebus/hosting/nservicebus-host/index_sample-initialization_host_[7,].partial.md

```diff
@@ -0,0 +1,19 @@
+
+The following snippet shows a sample configuration that can be used as a starting point.
+
+snippet: host-endpoint-config-sample
+
+### Set the host as startup project
+
+Either setup the start project to `NServiceBus.Host.exe` (available in the output directory of the build) or create a `launchSettings.json` with the following content
```

Comment:
```suggestion
Either set up the start project in Visual Studio to `NServiceBus.Host.exe` (available in the output directory of the build) or create a `launchSettings.json` for VS Code with the following content
```

Given the options are different per tool used, I've spelled out which version for what tool. Feel free to discard this suggestion if you think it's unnecessary.

Reply:
Actually launchsettings is also supported in Visual Studio. With the never version it even defaults to that.